api/config/module/bot/slackbot/v1/slackbot.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+
+package clutch.config.module.bot.slackbot.v1;
+
+option go_package = "github.com/lyft/clutch/backend/api/config/module/bot/slackbot/v1;slackbotv1";
+
+import "validate/validate.proto";
+
+message Config {
+  // Bot tokens begin with "xoxb-", https://api.slack.com/authentication/token-types#bot
+  string bot_token = 1 [ (validate.rules).string = {min_bytes : 1} ];
```

Comment:
Is there a chance we would want to have potentially n bot tokens if somehow we wanted a single clutch instance to talk to multiple slack teams or potentially have many bots?

Reply:
yeah it crossed my mind that it's something we might do in the future. though if we were to support that, then I think the config structure would be different. Perhaps it would be similar to Omnibot's https://lyft.github.io/omnibot/configuration.html#bots, where we'd want the bot app id, bot name, and bot token to know which messages to route to which bot.

I'll add a todo for now and come back to it when I open a PR for the module b/c may want to add more stuff

Reply:
👍  sounds good. My only concern would be that modifying the config down the road might be more difficult